.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 # . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged
+# npx lint-staged
```

Comment:
undo

Reply:
I had accidentally commented this out and committed. The original state was it was uncommented. 
_I had commented it because the linter previously was breaking the prisma code blocks that use the `view` block._

Reply:
There are comments/annotations on how to disable the linter I think, which would be the more permanent solution to that problem.